java/java-saltedge-example/src/com/saltedge/Program.java

```diff
@@ -15,4 +28,21 @@ public static void main (String[] arg) {
 //
 //        System.out.println(new SaltEdge().post("https://www.saltedge.com/api/v4/customers", jobject));
     }
+
+    private static void verifySignature() {
+        String data = CALLBACK_URL + "|" + CALLBACK_BODY;
+
+        try {
+            PublicKey publicKey = SignHelper.getPemPublicKey(PUBLIC_KEY_PATH, PUBLIC_KEY_ALGORITHM);
+            System.out.println("SIGNATURE BELONGS TO SALTEDGE: " + SignHelper.verify(data, SIGNATURE, publicKey));
+        } catch (IOException e) {
```

Comment:
you can do one catch with multiple exception types

Reply:
I think we should push the user to deal with these exceptions separately